gator/fragments.py

```diff
@@ -89,16 +84,14 @@ def is_blank_line(line):
 
 def count_paragraphs(contents):
     """Counts the number of paragraphs in the writing"""
-    replace_blank_inputs = True
-    matching_paragraphs = get_paragraphs(contents, replace_blank_inputs)
+    matching_paragraphs = get_paragraphs(contents)
     return len(matching_paragraphs)
 
 
 def count_words(contents):
     """Counts the minimum number of words across all paragraphs in writing"""
     # retrieve all of the paragraphs in the contents
-    replace_blank_inputs = False
-    paragraphs = get_paragraphs(contents, replace_blank_inputs)
+    paragraphs = get_paragraphs(contents)
```

Comment:
@gkapfham what was the use of `replace_blank_inputs` here? It was not clear from the source code exactly what mattered, and although I think now that we use an AST to parse the markdown it should not matter, I would like to confirm that with you.

Reply:
I don't see an obvious need for the `replace_blank_inputs` variable, @Michionlion. Did you ever discover a reason for this variable? It seems that your new approach obviates the need for it, right?

Reply:
That is correct, @gkapfham! I'll go with this implementation then.